.github/workflows/build-test-radosgw.yml

```diff
@@ -0,0 +1,77 @@
+---
+name: Build Radosgw
+on:
+  schedule:
+    - cron: '0 0 * * 0'  # refresh ccache every sunday night
+  pull_request:
+    branches:
+      - "s3gw"
+
+  push:
+    tags:
+      - "s3gw-v*"
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          path: ceph
+          submodules: recursive
+
+      - name: Create CCache Timestamp
+        id: ccache_timestamp
+        run: |
+          echo "::set-output name=timestamp::$(date +%Y-%m-%d-%H:%M:%S)"
+
+      - name: Cache CCache Files
+        uses: actions/cache@v3.0.4
+        with:
+          path: ceph/build.ccache
+          key: ccache-${{ steps.ccache_timestamp.outputs.timestamp }}
+          restore-keys: |
+            ccache-
+
+      - name: Build radosgw binary and tests
+        run: |
+          docker run --rm \
+            -v $GITHUB_WORKSPACE/ceph:/srv/ceph \
+            -e NPROC=4 \
+            -e WITH_TESTS=ON \
+            quay.io/s3gw/build-radosgw:latest
+
+      # - name: Run SFS unit tests
```

Comment:
is this meant to be commented by design? I don't remember seeing these comments in a prior version, but maybe I just overlooked them. If these are not needed, lets drop them please.